app/bundles/ChannelBundle/Command/SendChannelBroadcastCommand.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- * @copyright   2016 Mautic Contributors. All rights reserved
+ * @copyright   2019 Mautic Contributors. All rights reserved
```

Comment:
Let's not change the dates in the old files. 
```suggestion
 * @copyright   2016 Mautic Contributors. All rights reserved
```